### PR TITLE
mprven: Add a bit to enable MPRV to take effect in debug mode

### PR DESCRIPTION
**Pull request by mwachs5**
Addresses #240

### CODE REVIEW

**Comment by aswaterman** [comment]
lgtm

**Comment by timsifive** [comment]
Can we just remove the restriction that mprv is ignored in Debug Mode? That makes Debug Mode more like M Mode, and we don't need an extra bit at all.

**Comment by mwachs5** [comment]
That would be a major backwards incompatible change

File: core_debug.tex [hunk 1]
```diff
@@ -13,7 +13,12 @@ \section{Debug Mode} \label{debugmode}
 \begin{steps}{When executing code from the Program Buffer, the hart stays
     in Debug Mode and the following apply:}
 \item All operations are executed at machine mode privilege level, except that
-    \Fmprv in \Rmstatus is ignored.
+    \Fmprv in \Rmstatus may be ignored according to \Fmprven.
+    \begin{commentary}
+      In general, debugger SW is expected to be able to virtualize all the effects of \Fmprv.
```
Comment (timsifive) [inline core_debug.tex]: simulate instead of virtualize?

Comment (timsifive) [reply core_debug.tex]: "the debugger" instead of "debugger SW"

**Pull request merged**